android/app/src/main/java/org/chrome_on_device_ml/ml/QaClient.java

```diff
@@ -0,0 +1,228 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome_on_device_ml.ml;
+
+import android.content.Context;
+import android.content.res.AssetFileDescriptor;
+import android.content.res.AssetManager;
+import androidx.annotation.WorkerThread;
+import android.util.Log;
+import com.google.common.base.Joiner;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.ByteBuffer;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.tensorflow.lite.Interpreter;
+
+/** Interface to load TfLite model and provide predictions. */
+public class QaClient implements AutoCloseable {
+	private static final String TAG = "CDML_QaClient";
```

Comment:
We should make the spacing consistent across all the files. I think the files below are 2 spaces for tabs and I think thats the google style.

Reply:
@mcrouse-chrome thanks for the feedback. I changed all Java files to 2 spaces.